src/udp.rs

```diff
@@ -95,7 +102,14 @@ impl UdpSocket {
     }
 
     /// Receives data from the socket previously bound with connect(). On success, returns
-    /// the number of bytes read and the address from whence the data came.
+    /// the number of bytes read.
+    ///
+    /// The function must be called with valid byte array `buf` of sufficient size to
```

Comment:
I think we still need to mention that this function only writes into the buffer and never reads from it.